src/main/kotlin/tanvd/grazi/ide/GraziInspection.kt

```diff
@@ -3,89 +3,95 @@ package tanvd.grazi.ide
 import com.intellij.codeInsight.daemon.DaemonCodeAnalyzer
 import com.intellij.codeInspection.*
 import com.intellij.openapi.application.ApplicationManager
+import com.intellij.openapi.project.Project
 import com.intellij.openapi.project.ProjectManager
 import com.intellij.psi.PsiElement
 import com.intellij.psi.PsiElementVisitor
+import kotlinx.html.style
+import kotlinx.html.table
+import kotlinx.html.td
+import kotlinx.html.tr
 import tanvd.grazi.GraziConfig
 import tanvd.grazi.grammar.Typo
 import tanvd.grazi.ide.language.LanguageSupport
-import tanvd.grazi.ide.msg.GraziAppLifecycle
-import tanvd.grazi.ide.quickfix.*
+import tanvd.grazi.ide.msg.GraziStateLifecycle
+import tanvd.grazi.ide.quickfix.GraziAddWord
+import tanvd.grazi.ide.quickfix.GraziDisableRule
+import tanvd.grazi.ide.quickfix.GraziRenameTypo
+import tanvd.grazi.ide.quickfix.GraziReplaceTypo
+import tanvd.grazi.ide.ui.msg
 import tanvd.grazi.spellcheck.GraziSpellchecker
 import tanvd.grazi.utils.*
 import tanvd.kex.buildList
 
 class GraziInspection : LocalInspectionTool() {
-    companion object : GraziAppLifecycle {
+    companion object : GraziStateLifecycle {
         private fun getProblemMessage(fix: Typo): String {
             if (ApplicationManager.getApplication().isUnitTestMode) return fix.info.rule.id
+            return html {
+                if (fix.info.rule.description.length > 50 || fix.info.incorrectExample?.example?.length ?: 0 > 50) {
+                    style = "width: 300px;"
+                }
+
+                if (fix.isSpellingTypo) {
+                    +fix.info.rule.toDescriptionSanitized()
+                } else {
+                    table {
+                        attributes["cellpadding"] = "0"
```

Comment:
Add to HtmlUtils extension var to set this and `cellspacing` attribute

Reply:
Done